tools/docker/tests/test_flows.sh

```diff
@@ -82,8 +82,34 @@ test_ap_config_renew() {
 }
 
 test_ap_config_bss_tear_down() {
-    err "ap_config_bss_tear_down not implemented yet."
-    return 0
+    status "test ap config bss tear down"
+    # Regression test: MAC address should be case insensitive
+    MAC_AGENT1=$(echo $mac_agent1 | tr a-z A-Z)
+    # Configure the controller and send renew
+    send_CAPI_command gateway "DEV_RESET_DEFAULT" $redirect
+    send_CAPI_command gateway "DEV_SET_CONFIG,bss_info1,$MAC_AGENT1 8x Multi-AP-24G-1 0x0020 0x0008 maprocks1 0 1,bss_info2,$MAC_AGENT1 8x Multi-AP-24G-2 0x0020 0x0008 maprocks2 1 0" $redirect
```

Comment:
```suggestion
    send_CAPI_command gateway "DEV_SET_CONFIG,bss_info1,$mac_agent1 8x Multi-AP-24G-1 0x0020 0x0008 maprocks1 0 1,bss_info2,$mac_agent1 8x Multi-AP-24G-2 0x0020 0x0008 maprocks2 1 0" $redirect
```

And maybe a single BSS is sufficient too, so remove bss_info2.